### PR TITLE
Fixing a few broken links, testing new GH action

### PR DESCRIPTION
**Pull request by dylburger**

### CODE REVIEW

File: docs/docs/.vuepress/configs/sidebarConfig.js [hunk 1]
```diff
@@ -112,7 +104,16 @@ const referenceNav = [
     ],
   },
   "/limits/",
-  "/privacy-and-security/"
+  {
```
Comment (dylburger) [inline docs/docs/.vuepress/configs/sidebarConfig.js:107]: This was a small bug I found — security section wasn't rendering anything beyond `/privacy-and-security/` the way it was previously configured.

File: .eslintrc [hunk 2]
```diff
@@ -39,7 +39,9 @@
         "**/actions/**/common*.*js",
         "**/actions/common/*",
         "**/sources/common/*",
-        "**/sources/**/common*.*js"
```
Comment (dylburger) [inline .eslintrc:42]: These changes are unrelated, but I had to fix this to address some false positive errors Husky / ESLint raised in the `pre-commit` hook.

File: docs/docs/README.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ Pipedream is the fastest way to automate any process that connects APIs. Build a
 The Pipedream platform includes:
```
Comment (dylburger) [inline docs/docs/README.md:10]: Everything from here down is a change to a broken link, or removing a file tied to old docs.

File: docs/.gitignore [hunk 4]
```diff
@@ -1,3 +1,4 @@
 yarn-error.log
 node_modules
 .DS_Store
+docs/.vuepress/dist
```
Comment (ctrlaltdylan) [inline docs/.gitignore:4]: @dylburger funny, I added this to one of my more recent branches too. Got tired of `git clean`'ing every time.

File: docs/docs/.vuepress/configs/sidebarConfig.js (hunk 1 above)
Comment (ctrlaltdylan) [reply docs/docs/.vuepress/configs/sidebarConfig.js:107]: I was actually fixing this in another branch too, until I got hit with that refresh bug. I'll just clean up the merge conflict if there is one.

**Review by ctrlaltdylan — APPROVED** [review]
🛥️  it

**Pull request merged**